requirements.txt

```diff
@@ -47,6 +47,7 @@ shapely>=1.7.1
 pyclipper>=1.2.1
 psutil
 gdown
+basicsr
```

Comment:
could you please install packages related only to your tutorial inside tutorial in prerequisites section using following command:
`
!pip install -q basicsr
`